modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/ApplicationAPI.macro

```diff
@@ -0,0 +1,108 @@
+definition {
```

Comment:
How about name this file to ApplicationsAPI since it is headless-builder/applications in our API Explorer?

Reply:
good idea. the pattern you want to apply here sounds correct to me: to follow the *.macro structure the same way the API Applications in the headless builder are grouped. However, I'd leave it in singular here, as we want to place there methods that can be applied to the Application object (abstract). Does that make sense @yanan ? 😅

Reply:
clear, thanks for the confirmation!